src/Parser/GuardianDruid/Modules/Spells/Pulverize.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { formatPercentage } from 'common/format';
+import SpellIcon from 'common/SpellIcon';
+import SpellLink from 'common/SpellLink';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+import Module from 'Parser/Core/Module';
+import SPELLS from 'common/SPELLS';
+
+class Pulverize extends Module {
+
+  suggestions(when) {
+    const pulverizeUptimePercentage = this.owner.selectedCombatant.getBuffUptime(SPELLS.PULVERIZE_BUFF.id) / this.owner.fightDuration;
+    
+    this.owner.selectedCombatant.hasTalent(SPELLS.PULVERIZE_TALENT.id) && 
```

Comment:
This should be in on_initialized, setting this.active so that the entire module gets disabled properly.

Like so:
```jsx
  on_initialized() {
    if (!this.owner.error) {
      this.active = this.owner.selectedCombatant.hasTalent(SPELLS.PULVERIZE_TALENT.id);
    }
  }
```